user.js

```diff
@@ -675,49 +675,55 @@ module.exports = function user (options) {
 
       login.save$(function (err, login) {
         if (err) return done(err)
-        seneca.act({role: role, cmd: 'set_user_lock', id: user.id}, function (err, out) {
+        cmd_increment_lock(seneca, user.id, true, function (err, out) {
           if (err) return done(err)
+
           seneca.log.debug('login/ok', why, user, login)
           done(null, {ok: true, user: user, login: login, why: why})
```

Comment:
I think we should return user only after cleaning the information - take a look on `cleanUser` function.

I think that on `cleanUser` we should also remove the retry failed count information as we should not make it public/return it in the response.